tests/DqtApi.Tests/DataverseIntegration/CreateTeacherTests.cs

```diff
@@ -62,8 +62,18 @@ public async Task Given_details_that_do_not_match_existing_records_allocates_trn
             transactionRequest.AssertDoesNotContainCreateRequest<CrmTask>();
         }
 
-        [Fact]
-        public async Task Given_details_that_does_match_existing_record_does_not_allocate_trn_and_creates_QTS_task()
+        [Theory]
+        [InlineData(false, false, false, "")]
+        [InlineData(true, false, false, "Matched record has active sanctions\n")]
+        [InlineData(false, true, false, "Matched record has QTS date\n")]
+        [InlineData(false, false, true, "Matched record has EYTS date\n")]
+        [InlineData(true, true, false, "Matched record has active sanctions & QTS date\n")]
+        [InlineData(true, false, true, "Matched record has active sanctions & EYTS date\n")]
```

Comment:
```
        [InlineData(true, true, true, "Matched record has active sanctions & QTS date & EYTS date\n")]
```

There seems to be this hidden option.

Reply:
In reality that combination won't happen - it'll be at most one of EYTS or QTS.

Reply:
Can we just add a note for the next person, as `code vs reality` has a discrepancy.

Reply:
I've amended the implementation to better convey that `HasEytsDate` and `HasQtsDate` are mutually exclusive.